tune_sklearn/_trainable.py

```diff
@@ -143,20 +153,22 @@ def _train(self):
                 )
 
             ret = {}
-            for i, score in enumerate(scores["test_score"]):
-                key_str = f"split{i}_test_score"
-                ret[key_str] = score
-            self.test_accuracy = sum(scores["test_score"]) / len(
-                scores["test_score"])
-            ret["average_test_score"] = self.test_accuracy
+            for name in self.scoring:
+                for i, score in enumerate(scores["test_%s" % name]):
+                    key_str = f"split{i}_test_%s" % name
+                    ret[key_str] = score
+                self.test_accuracy = sum(scores["test_%s" % name]) / len(
+                    scores["test_%s" % name])
+                ret["average_test_%s" % name] = self.test_accuracy
```

Comment:
Hmm.. did we actually set `average_test_score` twice? This might've been something that slipped in our early implementation too

Reply:
Where is it set again? Am I missing something here

Reply:
Never mind, I expanded out the code and they're for the two different cases (with or without early stopping).. oops